src/auth/AuthServiceHandler.cc

```diff
@@ -26,6 +27,8 @@ AuthServiceHandler *get_auth_service_handler(int type, CephContext *cct, KeyServ
     return new CephxServiceHandler(cct, ks);
   case CEPH_AUTH_NONE:
     return new AuthNoneServiceHandler(cct);
+  case CEPH_AUTH_UNKNOWN:
+    return new AuthUnknownServiceHandler(cct);
```

Comment:
break what? sorry, I didn't get it. If the auth type neither cephx, none, nor unkonwn, it will still return null, is that your concern? thanks!